entities/Order.entity.ts

```diff
@@ -0,0 +1,114 @@
+import {
+  Entity,
+  Column,
+  PrimaryGeneratedColumn,
+  ManyToOne,
+  JoinColumn,
+  ManyToMany,
+  JoinTable,
+} from 'typeorm';
+import {
+  IsDefined,
+  IsUUID,
+  IsOptional,
+  Length,
+  IsDecimal,
+} from 'class-validator';
+import {
+  ApiHideProperty,
+  ApiProperty,
+  ApiPropertyOptional,
+} from '@nestjs/swagger';
+import { Auditable } from './Auditable';
+import { Company } from './Company.entity';
+import { User } from './User.entity';
+import { OrderStatus } from '../src/modules/order/order.enum';
+import { Item } from './Item.entity';
+
+@Entity('order')
+export class Order extends Auditable {
+  @PrimaryGeneratedColumn('uuid')
+  @IsOptional()
+  @IsUUID()
+  @ApiProperty({ description: 'Order Identifier' })
+  id?: string;
+
+  @Column()
+  @IsDefined()
+  @Length(2, 255)
+  @ApiProperty({
+    description: 'Status',
+    nullable: false,
+    type: String,
+    enum: OrderStatus,
+  })
+  status?: string;
```

Comment:
If we have an enum type for this field use it, if not, ignore this comment

Reply:
What do you mean about use it, so far the idea is that if an order is updated then we will send the corresponding value of the enum? or I am missing something?

Reply:
@jfernandezdev14 you defined status as an optional string in line 45, but I see in line 43 we documented it as an enum with 3 options, my point is, we should change the type on the line 45 from optional string to optional OrderStatus

Reply:
oohh I see, something like
´´´
status?: OrderStatus;
´´´
@matheo97 ?